docs/pages/project/contributing/contributing-error.md

```diff
@@ -140,9 +142,13 @@ var (
 )
```

Comment:
After the recent meshkit release we don't show panic error anymore so if you replace that part too it would be great.
Else current update LGTM.
![Screenshot from 2024-06-21 20-45-12](https://github.com/meshery/meshery/assets/99207534/f64fb0b2-7dba-46b0-84c3-81ac157f0ad4)

Try to run this code provided and see what does it show and please replace it if you would.

Reply:
Yes, right on the head that is the error that is displayed now so you can replace the panic error with the error you see and then we can be GTG with the PR.